src/components/ManagePost.vue

```diff
@@ -0,0 +1,230 @@
+<template>
+  <!-- TODO: Refactor so that existing posts can be passed in as props and re-use this interface -->
+  <div class="container col-6">
+    <button type="button" class="exit" @click="$emit('endManage')">X</button>
+    <form action="gotothedesiredURL" method="POST">
+      <input
+        v-model="post.title"
+        class="text-input"
+        placeholder="Write a title for your post"
+      />
+      <span id="privacySettings">
+        <input
```

Comment:
```suggestion
<input disabled
```
Disable PRIVATE option for now to avoid issues